v03_pipeline/lib/annotations/mito.py

```diff
@@ -79,9 +79,13 @@ def gt_stats(
         ref_samples_length = row.ref_samples[project_guid].length()
         heteroplasmic_samples_length = row.heteroplasmic_samples[project_guid].length()
         homoplasmic_samples_length = row.homoplasmic_samples[project_guid].length()
-        AC_het += heteroplasmic_samples_length # noqa: N806
-        AC_hom += homoplasmic_samples_length # noqa: N806
-        AN += (ref_samples_length + heteroplasmic_samples_length + homoplasmic_samples_length) # noqa: N806
+        AC_het += heteroplasmic_samples_length  # noqa: N806
```

Comment:
Have our `ruff` settings diverged somehow? If so.. can we put these lint changes on another pr?